[00:45:18] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[00:45:18] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[01:47:14] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[01:47:14] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[02:49:14] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[02:49:14] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[03:50:31] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[03:50:31] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[04:51:51] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[04:51:51] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[05:53:09] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[05:53:09] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[06:55:00] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[06:55:00] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[07:56:18] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[07:56:18] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[08:57:49] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[08:57:49] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[09:59:20] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[09:59:20] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[10:29:03] Error: Telegram API Error  { code: 4, http_status_code: 502, message: 'Bad Gateway' }
[10:29:03] Error: Telegram API long poll problem. Code: 4
Bad Gateway
[10:43:22] Error: Telegram API Error  {
  code: 4,
  http_status_code: 429,
  message: 'Too Many Requests: retry after 5'
}
[10:43:22] Error: Telegram API long poll problem. Code: 4
Too Many Requests: retry after 5
[11:00:56] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[11:00:56] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[12:02:46] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[12:02:46] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[13:04:57] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[13:04:57] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[14:06:27] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[14:06:27] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[15:08:39] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[15:08:39] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[16:10:49] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[16:10:49] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[17:12:17] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[17:12:17] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[18:14:22] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[18:14:22] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[19:16:16] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[19:16:16] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[19:28:24] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async VKBot.poll (/root/groupbot2/dist/vkbot.js:72:35)
[19:28:24] Error: {
  error_code: 10,
  error_msg: 'Internal server error',
  request_params: [Array]
}
[20:18:18] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[20:18:18] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[21:20:12] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[21:20:12] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[22:22:09] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[22:22:09] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
[23:24:21] Error: Error: An error has occurred:
    at VKBot.request (/root/groupbot2/dist/vkbot.js:54:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
[23:24:21] Error: {
  error_code: 901,
  error_msg: "Can't send messages for users without permission",
  request_params: [Array]
}
